src/app/components/OptimoPromos/Link/index.styles.jsx

```diff
@@ -0,0 +1,30 @@
+import { C_METAL, C_EBON } from '#app/legacy/psammead-styles/src/colours';
+import styled from '@emotion/styled';
+
+export default styled.a`
```

Comment:
```suggestion
const Link = styled.a`
```


`The definition name and filename should be identical. This makes it easier to locate code.`

You also easier to debug when components are named

Reply:
I will need to disable the following eslint rule [// eslint-disable-next-line import/prefer-default-export](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/prefer-default-export.md). 
I thought we were avoiding displaying rules on a file basis. Should I disable the rule entirely? 
(Dario TO DO)

Reply:
Just do this

```
export default Link;
```

Reply:
Done. 🤦‍♂️